src/udev/udev-builtin-net_id.c

```diff
@@ -1082,7 +1092,11 @@ static int get_link_info(sd_device *dev, LinkInfo *info) {
         if (r < 0 && r != -ENOENT)
                 return r;
 
-        (void) sd_device_get_sysattr_value(dev, "phys_port_name", &info->phys_port_name);
+        r = sd_device_get_sysattr_value(dev, "phys_port_name", &info->phys_port_name);
+        info->vf_representor_id = -1;
+        if (r >= 0)
+                /* Check if phys_port_name indicates virtual device representor */
+                sscanf(info->phys_port_name, "pf%*uvf%d", &info->vf_representor_id);
```

Comment:
if you ignore the result here, then make sure to cast this explicitly to `(void)` to make clear that you ignore the result here on purpose, for static checkers and such, since it's generally assumed you have to check the result of sscanf()

i.e.:

```c
(void) sscanf(info->…);
```

Reply:
Done... thx